### PR TITLE
FAQ update

### PR DESCRIPTION
**Pull request by kamild1996**
Mostly cosmetic changes. Also replaced nonfunctional :ref: links with standard href ones.

A native english speaker can probably improve this even more.

~squatDuck

### CODE REVIEW

File: docs/source/FAQ/index.rst [hunk 1]
```diff
@@ -67,73 +67,73 @@ Currently we don't have any special ranks in place.
...
+For a more detailed explanation about benefits of voting, please have a look at `this thread <https://mineyourmind.net/forum/threads/figuring-out-the-way-new-voting-tickets-and-tokens-work.13054/>`_.
 
 
 What protection does this server use?
 -------------------------------------
 
-On the general server's we use GriefPrevention due to its ease of use for the player. Other servers with modpacks like agrarian skies or crash landing use worldguard regions created from the skyblock plugin. Both third party protection systems are extended by our custom ProtectEx extension to protect against more modded items. More information about Grief Prevention can be found :ref:`here <ref-griefprevention-tutorial>`
+Generally we use GriefPrevention due to its ease of use for the player. Other servers with modpacks like Agrarian Skies or Crash Landing use WorldGuard regions created from the skyblock plugin. Both third party protection systems are extended by our custom ProtectEx extension to protect against more modded items. More information about Grief Prevention can be found `here <http://docs.mym.li/en/latest/Knowledge_Base/griefprevention.html>`_.
```
Comment (Slind14) [inline docs/source/FAQ/index.rst]: please keep the ref, else the link breaks once either the domain changes or the file.

File: docs/source/FAQ/index.rst [hunk 2]
```diff
@@ -205,9 +205,10 @@ Create a ticket requires a forum account which you can create `here <https://min
 
 How do I build a nether portal?
 ----------------------------------------
-See :ref:`Multiplex Nether Portal Tutorial <ref-nether-portal>`
+
+See: `Multiplex Nether Portal Tutorial <http://docs.mym.li/en/latest/Knowledge_Base/multiplex_nether_portal.html/>`_
```
Comment (Slind14) [inline docs/source/FAQ/index.rst]: same here, please keep the ref

**Comment by Slind14** [comment]
thank you for those improvements @kamild1996 :)
could you restore the references please, the direct links are not dynamic?

**Comment by kamild1996** [comment]
The problem with those refs were that they are not working, links are basically not appearing. I'll see what I can do.

**Comment by Slind14** [comment]
They were outdated, I fixed that.

**Comment by kamild1996** [comment]
Okay then, both dynamic links has been restored. Any more changes needed?

**Pull request merged**